CMakeLists.txt

```diff
@@ -318,7 +323,10 @@ if(NOT CMAKE_C_COMPILER_ID STREQUAL "Clang")
   endif()
 endif()
 
+# no this option for mwdt
+if(COMPILER STREQUAL gcc)
```

Comment:
Same comment as above, check not needed as the zephyr_cc_option_ifdef already verifies compiler flag support.

Reply:
Indeed, many compilers (not only gcc) may support -fstack-usage.